table/tables/index.go

```diff
@@ -177,7 +178,8 @@ func (c *index) GenIndexKey(indexedValues []types.Datum, h int64) (key []byte, d
 // Create creates a new entry in the kvIndex data.
 // If the index is unique and there is an existing entry with the same key,
 // Create will return the existing entry's handle as the first return value, ErrKeyExists as the second return value.
-func (c *index) Create(rm kv.RetrieverMutator, indexedValues []types.Datum, h int64) (int64, error) {
+func (c *index) Create(ctx context.Context, rm kv.RetrieverMutator, indexedValues []types.Datum, h int64) (int64, error) {
+	skipCheck := ctx.GetSessionVars().SkipConstraintCheck
```

Comment:
This is a dangerous operation. Should we disable the function of [tidb_skip_constraint_check](https://github.com/pingcap/tidb/blob/master/sessionctx/variable/tidb_vars.go#L39) variable.

Reply:
maybe we should use individual variable to indicate we are doing importing data?

Reply:
Yes, such as tidb_importing_data?

Reply:
But tidb_skip_constraint_check is skip handle check in https://github.com/pingcap/tidb/blob/f1e89cde1939ea88ec8927f8110da1b5a1b887f5/table/tables/tables.go#L410.